src/components/MapLocationDropdown.svelte

```diff
@@ -68,10 +68,9 @@
   };
 
   const checkSelectedValue = props => {
-    if (!selected) return;
     const nextSelected = findSelectedFromProps(props);
     if (selected !== nextSelected) {
-      selected = '';
+      selected = nextSelected ?? '';
```

Comment:
@ebrelsford I had to change a little bit of existing logic here which I think was either outdated or a bug, but want to make sure I'm not misunderstanding the reason this was written this way before!

Reply:
I see. I think the idea before was that this function only existed to clear the selected value once you move away from it. The way it works now means that if you wander around the map and somehow manage to match one of the options that will be selected for you. You can see that with the "zoomed out" scenario in the default config:
 1. Select "zoomed out"
 2. Zoom in once ("zoomed out" is no longer selected)
 3. Zoom out once ("zoomed out" is now selected)

I think that's fine but it might be slightly odd to have one of these options selected for you without you explicitly deciding to.